source/services/index.md

```diff
@@ -0,0 +1,105 @@
+An `Ember.Service` is a long-lived `Ember.Object` available application-wide. Services are singletons, and only one instance of a given service can exist at a time.
+
+Ember itself uses services to provide access to the router, and Ember Data does the same with the shared store.
+
+Other example uses of services include:
+
+* Logging
+* User/session authentication
+* Geolocation
+* Third-party APIs
+* Web Sockets
+* Server-sent events or notifications
+* Server-backed API calls that may not fit Ember Data
+
+Because `Ember.Service` doesn't add any sugar to `Ember.Object`, services are not required to follow as many conventions as components or routes. Anything that can be accomplished with an `Ember.Object` can be stored in a service.
+
+### Defining Services
+
+To define a service, extend the `Ember.Service` base class:
+
+```app/services/shopping-cart.js
+export default Ember.Service.extend({
+});
+```
+
+Like any `Ember.Object`, a service can have properties and methods of its own.
+
+```app/services/shopping-cart.js
+export default Ember.Service.extend({
+  items: [],
+
+  add(item) {
+    this.get('items').pushObject(item);
+  },
+
+  remove(item) {
+    this.get('items').removeObject(item);
+  },
+
+  empty() {
+    this.get('items').setObjects([]);
+  }
+});
+```
+
+### Accessing Services
+
+To access a service, inject it with `Ember.inject`:
+
+```app/components/cart-contents.js
+export default Ember.Component.extend({
+  cart: Ember.inject.service('shopping-cart'),
+});
+```
+
+You can then access properties and methods on the service:
+
+```app/components/cart-contents.js
+export default Ember.Component.extend({
+  cart: Ember.inject.service('shopping-cart'),
+
+  actions: {
+    remove(item) {
+      this.get('cart').remove(item);
+    }
+  }
+});
+```
+
+```app/templates/components/cart-contents.hbs
+<ul>
+  {{#each cart.items as |item|}}
+    <li>
+      {{item.name}}
+      <button {{action 'remove' item}}>Remove</button>
+    </li>
+  {{/each}}
+</ul>
+```
+
+The injected property is lazy; the service will not be instantiated until the property is explicitly called. It will then persist until the application exits.
+
+If no argument is provided to `service()`, Ember will use the dasherized version of the property name:
+
+```app/components/cart-contents.js
+export default Ember.Component.extend({
+  shoppingCart: Ember.inject.service()
+});
+```
+
+### Injecting Services Eagerly
```

Comment:
I'd like to see some discussion on the inclusion of this section. cc @michaelrkn